### ISSUE TITLE
Vector chokes on GIFs sent by mubot

### ISSUE THREAD

**Issue by matrixbot** [vector-im/vector-web#475]
Created by @ matthew:matrix.org.

**Comment by erikjohnston** [comment]
For example:

```
{
  "origin_server_ts": 1449405928405,
  "user_id": "@mubot:matrix.org",
  "event_id": "$144940592877511pIdrQ:matrix.org",
  "content": {
    "body": "http://i.imgur.com/zumSlIA.gif",
    "url": "mxc://matrix.org/nuYiaqOswAGtGdIxENmtrAQj",
    "msgtype": "m.image"
  },
  "room_id": "!cURbafjkfsMDVwdRDQ:matrix.org",
  "type": "m.room.message",
  "age": 1946
}
```

Note lack of any metadata about image.

Clicking gives error in console:

```
Uncaught TypeError: Cannot read property 'w' of undefined
e.exports.r.createClass.onClick @ MImageBody.js:59
....
```

**Comment by davidar** [comment]
Also note that this works fine in android console.

**Comment by ara4n** [comment]
The NPE itself is fixed in 072130466cfa38a9431361ec74f8449011f08c00 on matrix-react-sdk and 8ecf70dda0fbbaf21e07a9c10edc2aab0c685872 on vector.
